Mathlib/Algebra/Group/Subgroup/Basic.lean

```diff
@@ -351,12 +351,29 @@ theorem normalizer_eq_top [h : H.Normal] : H.normalizer = ⊤ :=
   normalizer_eq_top_iff.mpr h
 
 @[to_additive]
-theorem le_normalizer_of_normal [hK : (H.subgroupOf K).Normal] (HK : H ≤ K) : K ≤ H.normalizer :=
+theorem subset_normalizer_of_normal {S : Set G} [hH : H.Normal] : S ⊆ H.normalizer :=
+  normalizer_eq_top H ▸ le_top
+
+/- TODO: rename `le_normalizer_of_normal'` -> `le_normalizer_of_normal` when the current deprecation
+`le_normalizer_of_normal` -> `le_normalizer_of_normal_subgroupOf` is completed.
+-/
+@[to_additive]
+theorem le_normalizer_of_normal' [hH : H.Normal] : K ≤ H.normalizer := subset_normalizer_of_normal
+
+@[to_additive]
+theorem le_normalizer_of_normal_subgroupOf [hK : (H.subgroupOf K).Normal] (HK : H ≤ K) :
```

Comment:
Is it possible for this PR to wait for #22913? I feel that there's some cleanup of these normalizer lemmas that should happen first.

Reply:
Ok, thanks. It's merged now.